fileWithPRComments.js

```diff
@@ -0,0 +1,8 @@
+/* global QamineUtils */
+
+"use strict";
+
+(function($, window) {
+  $('.jsSelectedMainEmail').text(postVariable.email);
```

Comment:
Codacy found an issue: ['postVariable' is not defined.](https://app.dev.codacy.org/gh/devcodacytestuser/CheckPrCommentsPublicProjectDev/pullRequest?prid=23615)